table/keys.go

```diff
@@ -30,44 +30,84 @@ type KeyMap struct {
 	ScrollLeft key.Binding
 }
 
-// DefaultKeyMap returns a set of sensible defaults for controlling a focused table.
+// DefaultKeyMap returns a set of sensible defaults for controlling a focused table with help text.
 func DefaultKeyMap() KeyMap {
 	return KeyMap{
 		RowDown: key.NewBinding(
 			key.WithKeys("down", "j"),
+			key.WithHelp("↓/j", "move down"),
 		),
 		RowUp: key.NewBinding(
 			key.WithKeys("up", "k"),
+			key.WithHelp("↑/k", "move up"),
 		),
 		RowSelectToggle: key.NewBinding(
 			key.WithKeys(" ", "enter"),
+			key.WithHelp("<space>/enter", "select row"),
 		),
 		PageDown: key.NewBinding(
 			key.WithKeys("right", "l", "pgdown"),
+			key.WithHelp("→/h/page down", "Previous Page"),
 		),
 		PageUp: key.NewBinding(
 			key.WithKeys("left", "h", "pgup"),
+			key.WithHelp("←/h/page up", "Next Page"),
```

Comment:
```suggestion
			key.WithHelp("←/h/page up", "Previous Page"),
```

Reply:
Fixed these in newest commits. Just testing CI/CD locally now if everything passes then I commit the changes :)